app/webpacker/components/Schedule/TimeZone.jsx

```diff
@@ -46,7 +46,7 @@ export default function TimeZoneSelector({
         search
         selection
         value={activeTimeZone}
-        onChange={(_, data) => setActiveTimeZone(data.value)}
+        onChange={(__, data) => setActiveTimeZone(data.value)}
```

Comment:
Calling this `_e` should also work, and is at least not quite as "silent" as your double-underscore approach.